examples/custom-lambda/main.tf

```diff
@@ -0,0 +1,12 @@
+module "notify_slack" {
```

Comment:
Please more this module call into `examples/notify-slack-simple` for brevity, and verify it by running the whole example (with standard and custom source code).

Reply:
Done. I've moved it to `examples/notify-slack-simple/custom-lambda.tf` and I've test it as you suggested.